### ISSUE TITLE
IBEX menu hidden and can't get it back

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#3182]
A creative POLARIS user managed to press some (unknown) keyboard shortcuts which got the IBEX client into a state where the top menu bar was not visible.

See screenshot below:

![image](https://user-images.githubusercontent.com/24375813/40050338-0c271a8c-582f-11e8-8171-fd631f3c3e29.png)

As part of this ticket:
- Experiment with various keyboard shortcuts and see if you can get the client into the state above
- Document how to get out of this state on the wiki

For the immediate issue, I have told Ron to reopen a new client when the current script is finished. I checked that the new client opens sensibly.

**Comment by kjwoodsISIS** [comment]
Users can always use Alt+F4 to close any window.  They can simply close the IBEX client with Alt+F4 and then re-start.

**Comment by Tom-Willemsen** [comment]
Yes that's correct, they could close it, in this case they didn't want to close it because they had a script running. I had a go at getting the menu back for a few minutes (to no avail) and then just told Ron to reopen a new client once the script was finished.

**Comment by Tom-Willemsen** [comment]
To reproduce (we think):
* Right -click in OPI
* Select "Compact Mode"

The only way to get out of this is to right click an OPI again and select "Exit Compact Mode". 

This gets worse if you right click and enter full screen.

**Comment by KathrynBaker** [comment]
If you use right-click and enter full screen mode it also hides the windows explorer bar at the bottom as well as the IBEX menu bar (seen on MERLIN), and using those in any combination can do some nasty things which reset layout does not resolve.

**Comment by KathrynBaker** [comment]
This situation has changed, Full Screen and Exit still seems to behave, but `exit compact mode` wasn't restoring that menu in the built client installed on my desktop

**Comment by KathrynBaker** [comment]
Straightforward, but potentially frustrating if you're not using the script server - but not exactly simple to get into the state, so medium is fine at the moment

**Comment by rerpha** [comment]
Noticed this on IMAT the other day, but as said above restarting fixed it

**Comment by KathrynBaker** [comment]
Still in the same state as February 2022, and still causing occasional confusion
